### DISCUSSION TITLE
Differential rates for particular project types

### DISCUSSION THREAD

**Discussion by JeremySilver** [kimai/kimai#5182] [General] [answered]
Hi there,

I was wondering whether Kimai supports differential rates within particular project types? For example, the first 5 hours get billed at rate A, then the subsequent 10 hours get billed at rate B, then any subsequent hours get billed at rate C? If so, is it simple to apply to projects when setting them up?

Thanks!

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/documentation/rates.html

After reading that one, I would say: No.

**Reply by JeremySilver** [reply]
Right - thanks for confirming that.
